gta3_sbl/classes.db

```diff
@@ -1204,7 +1210,7 @@ RegisterAmbulanceLevel,0403,0,0,("highestLevel%i")
 RegisterCriminalCaught,0402,0,0,()
 RegisterDefuseBombTime,0407,0,0,("timeInSec%i")
 RegisterElBurroTime,03E2,0,0,("timeInSec%i")
-RegisterFastestTime,042E,0,0,("statId%i" "value%i")
+RegisterFastestTime,042E,0,0,("id: StatId" "value%i")
```

Comment:
Hm it was added to gta3? There is no StatId enum for that version

Reply:
https://library.sannybuilder.com/#/gta3/default/042E
https://library.sannybuilder.com/#/vc/default/042E

want to add StatId enum in III/VC?